server/app/routes.py

```diff
@@ -13,3 +19,20 @@ def videos_raw():
 	@app.route('/videos')
 	def videos():
 		return jsonify(rss.summarize_watchlist(app.config['watchlist']))
+
+	@app.route('/watchlist', methods=['POST', 'DELETE'])
+	def modify_watchlist():
+		# only required value is 'id'
+		json = request.get_json()
+		if 'id' not in json:
+			return 'invalid json'
```

Comment:
need to return the appropriate http code (probably 400)

Reply:
done